examples/spring-boot-starter-example-jakarta/pom.xml

```diff
@@ -0,0 +1,48 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.moesif.servlet.spring</groupId>
+  <artifactId>spring-boot-starter-example-jakarta</artifactId>
+  <name>spring-boot-starter-example-jakarta</name>
+  <packaging>war</packaging>
+  <version>6.0-beta-1</version>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.0.0</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    <version>3.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.servlet</groupId>
+      <artifactId>jakarta.servlet-api</artifactId>
+      <version>6.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.tomcat.embed</groupId>
+      <artifactId>tomcat-embed-jasper</artifactId>
+      <scope>default</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.moesif.servlet</groupId>
+      <artifactId>moesif-servlet-jakarta</artifactId>
+      <version>2.0-beta-2</version>
```

Comment:
@JideoforNwoke This `jar` is currently available in maven central repository if you would like to try it.

Reply:
@CodeYouMust 
Alright,
Thank you